lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -324,6 +390,7 @@ public void setAnimation(String animationName) {
    */
   public void setAnimation(final String animationName, final CacheStrategy cacheStrategy) {
     this.animationName = animationName;
+    animationResId = -1;
```

Comment:
Set this to 0.

Reply:
Let's stick to 0 for an invalid resource then :)